### PR TITLE
Simplify rpm install verification in improved sanity test

### PR DESCRIPTION
**Pull request by mike-nguyen**
Addresses issue #175.  The previous method of verifying rpm installs
relied scraping HTML to determine the filename for a RPM before it
could install it.  There was also a single point of failure pointing
to dl.fedoraproject.org instead of the mirror site.

This PR simplifies the verification in the improved sanity test by
using the epel-release package in the mirror site which always has
the same URL.

### CODE REVIEW

File: roles/rpm_install_epel/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+# vim: set ft=ansible:
+#
+# This role will install epel7 from download.fedoraproject.org and
+#  installs it using the rpm command.  This hsould be used after
```
Comment (miabbott) [inline roles/rpm_install_epel/tasks/main.yml]: s/hsould/should/

**Comment by miabbott** [comment]
Instead of a new custom role, I'd rather see the `rpm_install` role reworked to perform an install from a URL.  Then, if we were in the situation where we have to perform some additional work to derive a URL, we could make a separate role to do that.

I'd do something just like this:

```yaml
---                                                                                                                                                                                                                
# vim: set ft=ansible:                                                                                                                                                                                             
#                                                                                                                                                                                                                  
- name: Fail if rpm_url is not defined                                                                                                                                                                             
  fail:                                                                                                                                                                                                            
    msg: "rpm_url must be passed into this role/include"                                                                                                                                                           
  when: rpm_url is undefined                                                                                                                                                                                       
                                                                                                                                                                                                                   
- name: Install RPM from URL                                                                                                                                                                                       
  command:  rpm -i {{ rpm_url }}
```

It would require some more changes to `tests/admin-unlock/main.yml` but there is no reason we can't just use the `epel-release` package again in that test.

WDYT?

**Comment by mike-nguyen** [comment]
Makes too much sense to me.

**Comment by mike-nguyen** [comment]
bot, retest this please

**Comment by miabbott** [comment]
LGTM!

**Pull request merged**